docs/write_test/guidelines.rst

```diff
@@ -8,6 +8,7 @@ Coding guidelines
 -  `Logging <#logging>`__
 -  `Error message <#error-message>`__
 -  `Assertion <#assertion>`__
+-  `Tips for writing code <#tips-for-writing-code>`__
```

Comment:
Move it to lower priority, just before the "tips for non-native..."

Reply:
updated